runner.js

```diff
@@ -49,8 +49,8 @@ module.exports = function (config) {
   var preview = function (sysDef, name, cb) { previewer.preview(sysDef, name, cb) }
   var previewAll = function (sysDef, cb) { previewer.previewAll(sysDef, cb) }
 
-  var grep = function (name, config, search, cb) { util.grep(name, config, search, cb) }
-  var grepAll = function (system, config, search, cb) { util.grepAll(system, config, search, cb) }
+  var grep = function (name, search, cb) { util.grep(name, search, cb) }
+  var grepAll = function (system, search, cb) { util.grepAll(system, search, cb) }
```

Comment:
yep - pending PR from @calebissharp that cleans up this file - will merge once this is in!